addon/components/horizontal-bar-chart.js

```diff
@@ -116,7 +116,21 @@ export default ChartComponent.extend(FloatingTooltipMixin,
   }),
 
   marginTop: Ember.computed.alias('labelPadding'),
-  marginBottom: Ember.computed.alias('labelPadding'),
+
+  /**
+   * The margin at the bottom depends on the label and title padding and height.
+   * @override
+   * @type {Number}
+   */
+  marginBottom: Ember.computed('labelPadding', 'xTitleVerticalOffset',
+      'hasXAxisTitle', function() {
+    if (this.get('hasXAxisTitle')) {
+      // 15 is the expected height of the text.
+      return this.get('labelPadding') + this.get('xTitleVerticalOffset') + 15;
```

Comment:
Just curious, when you say the "expected height of the text" are you making an assumption about the default font size and taking the height of the text given the default?